src/dashboard/index.ts

```diff
@@ -10,9 +10,33 @@ let mainWindow: BrowserWindow | null
 let node: Node | null
 let firefox: Firefox | null
 
+let isFirefoxRunning = false
+let isLoggingOut = false
+
 declare const DASHBOARD_WINDOW_PRELOAD_WEBPACK_ENTRY: string
 declare const DASHBOARD_WINDOW_WEBPACK_ENTRY: string
 
+const MESSAGES = {
```

Comment:
I thought it would be nice to consolidate all user-facing texts in one place. At the moment, they're in the same file they are used, but we could discuss the possibility of putting them somewhere else, to make it easy to make quick updates to texts and eventually load them from somewhere else if we need to deal with translations.

Reply:
I like this idea. I remember that in PHP I used to have these `i18n` (internationalization) and `l11n` (localization) files, with all the message strings. Good ol' days.

We could use something like https://www.i18next.com/